templates/emails/token_request.txt

```diff
@@ -0,0 +1,17 @@
+Dear REANA administrator,
+
+New user access token request:
+
+{{ user_data }}
+
+To grant the user token, please run:
+
+$ kubectl exec -ti reana-server-xxx -- flask reana-admin token-grant -e {{ user_email }} --admin-access-token $REANA_ACCESS_TOKEN
```

Comment:
`kubectl exec -ti reana-server-xxx -- flask...` is not copy-paste friendly.

What about using `kubectl exec -i -t deployment/reana-server -- flask...` instead?

Reply:
(BTW, we have `token-grant` but then `users-create`... Would be nice to harmonise *later* to use singular or plural consistently everywhere.)